python/strings_udf/cpp/src/strings/udf/udf_apis.cu

```diff
@@ -42,7 +42,8 @@ namespace {
 struct udf_string_to_string_view_transform_fn {
   __device__ cudf::string_view operator()(cudf::strings::udf::udf_string const& dstr)
   {
-    return cudf::string_view{dstr.data(), dstr.size_bytes()};
+    return dstr.data() == nullptr ? cudf::string_view{"", 0}
+                                  : cudf::string_view{dstr.data(), dstr.size_bytes()};
```

Comment:
```suggestion
    return dstr.data() == nullptr ? cudf::string_view{}
                                  : cudf::string_view{dstr.data(), dstr.size_bytes()};
```

I think `string_view` has a default empty constructor.

Reply:
> I think `string_view` has a default empty constructor.

You are correct.